### ISSUE TITLE
Accessibility -  Information Screens - TalkBack Does Not Read Content

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Child-Labor-Android#21]
While attempting to read the ‘Introduction’, ‘Reports Fact Sheet’, ‘OCFT fact Sheet’, and ‘Programs Fact Sheet’ Screens with TalkBack the only thing read was the ‘back button” and the word screen name. Talkback never read any of the other content, which I’m guessing is a PDF.

**Comment by aquilent-dgemoets** [comment]
Smarter versions of the PDFs are dependent on a contractor and are not available yet.  It's a deficiency of both the Android and iOS versions; at least we've attained parity with the live iOS version.

That said, as discussed in Issue #5, if time permits we may be able to add a hook to ask the Android phone to open the PDF using software already installed on the phone, which could include a TalkBack-compatible PDF reader.

**Comment by aquilent-dgemoets** [comment]
ILAB has re-prioritized this issue and we will try to add a hook to ask the Android phone to open the PDF using software already installed on the phone.

**Comment by jfc3-dol** [comment]
The PDF's in the "more Information" section are still not being read by TalkBack.

**Comment by aquilent-dgemoets** [comment]
The latest beta (uploaded Friday 12/4) includes the ability to have the Android phone open the PDF using software already installed on the phone (such as one that reads PDFs via TalkBack).

**Comment by nmangu** [comment]
Update after the BETA 5 Release: Talkback still Not Reading the PDF contents.

**Comment by aquilent-trumainejohnson** [comment]
Unfortunately Android does not provide a native PDF reader that is accessible like IOS.  Because of this we have to rely on third party alternatives, which I choose the best open source one I could find.  

As an alternative to using the built in pdf reader which is not accessible and has blurry text you can click on the "Share Icon" in the upper right corner when viewing a pdf to open it in an external application.

I would recommend using the official Adobe Acrobat Reader (https://play.google.com/store/apps/details?id=com.adobe.reader&hl=en).  It provides a better viewing experience (No blurry text) and is also accessible (TalkBack will read the PDF content).

**Comment by nmangu** [comment]
Will try to verify with the 'Share Icon' and update.

**Comment by nmangu** [comment]
BETA-5 Release Update: PDF using 'Share Icon' is reading the page contents in the 'Drive PDF Viewer' (external application). No blurry text found while navigating through the pages (in this case used country Brazil).  Verified and validated as working.

**Comment by jfc3-dol** [comment]
The application is still not reading the PDFs within the application.

**Comment by jfc3-dol** [comment]
These items don't seem be in the application anymore but TalkBack is not reading the PDF content.

**Comment by carolynhuang** [comment]
Close out on this issue. Fact sheets no longer exist in the app.

**Comment by jfc3-dol** [comment]
The fact sheet is no longer there but all the other PDFs under the information section such as the "Reports" and "FAQs" section items and the "An Intro to OCFT" are not being read correctly.

The files might have changed, but the same issue applies.

I can close this one issue and open one up for each of the other issues if you like.

**Comment by jfc3-dol** [comment]
Opened issues #66 that pertains to not reading PDFs properly, When this issue was about not reading them at all.
